client/hbs/about.hbs

```diff
@@ -1,29 +1,44 @@
-<strong>Down to the Wire</strong> is a collaborative project hosted by a group of students across the country. Since we share many common interests, we decided to merge all of our individual blogs and technical writings into a single website, where all of us can contribute. Since each author has different interests and personal projects, topics range from hardware to software to security.  
+<html>
+{{> head.hbs}}
 
-<table id="profiles" class="row-divided">
-{{#each (profiles)}}
-{{#if this.bio}}
-	<tr>
-		<th><a href="/author/{{this.handle}}" name="{{this.handle}}"><img class="author-image large" src="/images/{{#if this.image}}{{this.image}}{{else}}default.png{{/if}}"/></a></th>
-		<td>
-			<div class="bio">{{this.bio}}</div>
-			<div class="contact">
-				<address>
-					<a href="mailto:{{this.handle}}@dttw.tech">{{this.handle}}@dttw.tech</a>
-				</address>
-				<address>
-					{{#if this.website}}
-					<a href="{{this.website}}">{{this.website}}</a>
-					{{/if}}
-				</address>
-				<address>
-					{{#if this.github}}
-					<a href="mailto:{{this.handle}}@dttw.tech">{{this.handle}}@dttw.tech</a>
+<body>
+	{{> header.hbs}}
+	<content-row>
+		<main>
+			<strong>Down to the Wire</strong> is a collaborative project hosted by a group of students across the country. Since we share many common interests, we decided to merge all of our individual blogs and technical writings into a single website, where all of us can contribute. Since each author has different interests and personal projects, topics range from hardware to software to security.
+
+			<table id="profiles" class="row-divided">
+				{{#each (profiles)}}
+					{{#if this.bio}}
+						<tr>
+							<th>
+								<a href="/author/{{this.handle}}" name="{{this.handle}}"><img class="author-image large" src="/images/{{#if this.image}}{{this.image}}{{else}}default.png{{/if}}" /></a>
+							</th>
+							<td>
+								<div class="bio">{{this.bio}}</div>
+								<div class="contact">
+									<address>
+										<a href="mailto:{{this.handle}}@dttw.tech">{{this.handle}}@dttw.tech</a>
+									</address>
+									<address>
+										{{#if this.website}}
+											<a href="{{this.website}}">{{this.website}}</a>
+										{{/if}}
+									</address>
+									<address>
+										{{#if this.github}}
+											<a href="mailto:{{this.handle}}@dttw.tech">{{this.handle}}@dttw.tech</a>
+										{{/if}}
+									</address>
+								</div>
+							</td>
+						</tr>
 					{{/if}}
-				</address>
-			</div>
-		</td>
-	</tr>
-{{/if}}
-{{/each}}
-</table>
+				{{/each}}
+			</table>
+		</main>
+		{{> sidebar.hbs}}
```

Comment:
So what's the reasoning for switching from a frame containing the auxiliary pages (like `about` and such) to a standalone page that pulls in the components it needs. I tend to prefer the former because it allows more easy extensibility.